docs/introduction/QuickStart.md

```diff
@@ -0,0 +1,200 @@
+
+```js
+const store = configureStore({
+  reducer: calculatorSlice.reducer
+})
+```
+
+Using Redux Toolkit's `configureStore`, we can setup the store with helpful default middleware out of the box. Unfortunately, this example won't be able to highlight this middleware, but be sure to check out the [included default middleware](https://redux-starter-kit.js.org/api/getDefaultMiddleware#included-default-middleware) to learn more about them.
+
+To Recap, here's the full code that we've written so far:
+
+```js
+const calculatorSlice = createSlice({
+  name: 'calculator',
+  initialState: 0,
+  reducers: {
+    add: (state, action) => state + action.payload,
+    subtract: (state, action) => state - action.payload
+  }
+})
+
+const store = configureStore({
+  reducer: calculatorSlice.reducer
+})
+```
+
+Not too bad :)
+
+## Vanilla Example
+
+Great! Now that we have the redux portion of the code done, let's move onto creating an application that can utilize it.
+
+```js
+const valueEl = document.getElementById('value')
+
+const render = function() {
+  valueEl.innerHTML = store.getState().toString()
+}
+
+// we'll need to render it once to populate the value element with some data
+render()
+
+// Setup the subscription to the value element
+store.subscribe(render)
+
+// let's destructure the actions from the calculator slice
+const { add, subtract } = calculatorSlice.actions
+
+// Setup event listeners for the buttons
+document.getElementById('add').addEventListener('click', function() {
+  const value = document.getElementById('input').value
+  store.dispatch(add(value))
+})
+
+document.getElementById('subtract').addEventListener('click', function() {
+  const value = document.getElementById('input').value
+  store.dispatch(subtract(value))
+})
+```
+
+alright, now we have a simple app setup that utilizes our Redux code to have a functioning calculator app. it will now dispatch actions to update the value in the store,
+depending on which button was clicked.
+
+## React Example
+
+Now let's take a look at the same code, but built in React. We'll need to use the `react-redux` package for the official React bindings of the Redux store.
+
+We'll also be using the new React Redux hooks API, so be sure to check that out if you haven't already.
+
+```jsx
+const Value = function() {
+  const value = useSelector(state => state)
+
+  return <span>{value}</span>
+}
+
+const Button = function(props) {
+  const { action, children } = props
+  const dispatch = useDispatch()
+
+  return <button onClick={() => dispatch(action)}>{children}</button>
+}
+
+const App = function() {
+  const [input, setInput] = useState(0)
+
+  const handleChange = function(e) {
+    const { value } = e.target
+    setInput(Number(value))
+  }
+
+  return (
+    <Provider store={store}>
+      The value is <Value />
+      <input onChange={handleChange} />
+      <Button action={add(input)}>Add Amount</Button>
+      <Button action={subtract(input)}>Subtract Amount</Button>
```

Comment:
This looks *bad*, I'm not sure why. It almost makes this code snippet unusable, since if I'm an end user copy/pasting this code I would most likely re-write this portion entirely.